### PR TITLE
Sidebar Info

### LINKED ISSUE #3: Pokemon Image and Basic Information
**Is your feature request related to a problem? Please describe.**
This feature will contain the initial shown information about the pokemon and its picture

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Initialization of the sidebar functionality 

## Purpose
To allow users to see more information about the selected pokemon

## Approach
Created state to hold the selected pokemon and passed it to the required siblings. 

Closes #3

### CODE REVIEW

**Review by BlackCubes — APPROVED** [review]
Looks gewd Green Squad!

**Review by Lap343 — APPROVED** [review]
Looks good to me

File: pokeapp/src/components/Sidebar.js [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+      return <p key={index}>{abilityObject.ability.name}</p>;
+    });
+  };
+
+  const gameMapper = (games) => {
+    return games?.map((gameObject, index) => {
+      return <p key={index}>{gameObject.version.name}</p>;
+    });
+  };
+
```
Comment (JorgeLVilla) [inline pokeapp/src/components/Sidebar.js:28]: Excited to this in action!

**Review by JorgeLVilla — APPROVED** [review]
Looks great!

**Review by Gjhnsn — APPROVED** [review]
Looking goooood!

**Pull request merged**